src/cljs/athens/views/athena.cljs

```diff
@@ -205,10 +214,24 @@
       ;; TODO: change scroll as user reaches top or bottom
       ;; TODO: what happens when user goes to -1? or past end of list?
       (= key KeyCodes.UP)
-      (swap! state update :index dec)
+      (do
+        (swap! state update :index #(dec (if (zero? index) (count results) index)))
+        (let [cur-index (:index @state)
+              input-el (.. e -target)
+              result-el (.. (. input-el closest "div.athena") -lastElementChild)
```

Comment:
What does this do? add comment?

Reply:
also you are already using `..` syntax sugar, so it should just be
```(.. input-el (closest "div.athena") -lastElementChild)```
recommend http://app.klipse.tech/

Reply:
> What does this do? add comment?

Yeah looking at it now, it's not obvious why I'm doing that. I'll add some comments 👍

Reply:
> also you are already using `..` syntax sugar, so it should just be
> `(.. input-el (closest "div.athena") -lastElementChild)`
> recommend http://app.klipse.tech/

This is awesome! Thanks for the link.